generate/struct.go

```diff
@@ -26,18 +26,26 @@ type extractedField struct {
 // write outputs the Golang representation of this field to the writer with prefix before each line.
 // It handles inline structs by calling this method recursively adding a new \t to the prefix for each layer.
 // If required is set to false 'omitempty' is added in the JSON struct tag for the field
-func (ef *extractedField) write(w io.Writer, prefix string, required bool) error {
+func (ef *extractedField) write(w io.Writer, prefix string, required, descriptionAsStructTag bool) error {
 	var omitempty string
 	if !required {
 		omitempty = ",omitempty"
 	}
 	jsonTag := fmt.Sprintf(`json:"%s%s"`, ef.jsonName, omitempty)
 	var description string
-	if ef.description != "" {
+	if descriptionAsStructTag && ef.description != "" {
 		description = fmt.Sprintf(`description:"%s"`, strings.Split(ef.description, "\n")[0])
```

Comment:
I know this is existing code but is there any value to not splitting the description on a newline? Seems strange to me that if there's any newlines at all the rest will be omitted

Reply:
It is strange, it took me a couple of tries to get the test files right because of this. If you have a newline in the description it will mess up the go struct that is built so they need to be stripped. I'm guessing when it was first implemented rather than just replacing them with spaces and making a huge description it was considered better to just get the first line. I didn't fix it because I didn't want to change any existing behavior (or really I didn't want to update the test data :)

Reply:
:joy: Yeah I figured it would break the struct tags but I would much prefer we replace newlines with spaces. I haven't looked into it but how many test files would actually need updating? I don't think we really use that many newlines in descriptions in our actual schema definitions (not test files)